core/src/test/scala/org/apache/spark/sql/store/SnappyCatalogSuite.scala

```diff
@@ -413,7 +413,10 @@ abstract class CatalogTestUtils {
         CatalogColumn("a", "int"),
         CatalogColumn("b", "string")),
       partitionColumnNames = Seq("a", "b"),
-      bucketColumnNames = Seq("col1"))
+      bucketColumnNames = Seq("col1"),
+      // Pass the properties so that the hack applied in SnappyExternalCatalog.createTable
+      // takes affect
+      properties = Map("spark.sql.sources.provider" -> "column"))
```

Comment:
Well, I was trying to fix the failures in SnappyCatalogSuite (both with 2.0.0 and 2.0.2) and thought it could be the reason. But it isn't it. But kept it here as it takes it to the correct block which should be tested.